File: Java/UberBlack.java
public class UberBlack extends Car {
    
    String typecaraccepted;
    String seatsmaterial;
    public UberBlack(String license, Account driver, String typecaraccepted, String seatsmaterial) {
        super(license, driver);
        this.typecaraccepted = typecaraccepted;
        this.seatsmaterial = seatsmaterial;
    }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    #La herencia de la familia car
    typeCarAccepted = []
    def __init__(self, license, driver, typeCarAccepted, seatsMaterial) -> None:
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial